buku.1

```diff
@@ -39,6 +39,7 @@ URLs are unique in DB. The same URL cannot be added twice. You can update tags a
   - --st : search bookmarks by tag, or show all tags alphabetically.
   - You can search bookmarks by tag (see examples below).
   - Search results are indexed serially. This index is different from actual database index of a bookmark record which is shown in bold within '[]' after the URL.
+  - Can delete the search results, When any any above search flag is combined with -d.
```

Comment:
I guess this doesn't work with `--st` without any keyword, because that will simply list the tags. This needs to be re-worded. Also, this should go in the **Operational notes** section of the readme.